src/ray/gcs/gcs_server/gcs_placement_group_manager.h

```diff
@@ -169,6 +169,9 @@ class GcsPlacementGroupManager : public rpc::PlacementGroupInfoHandler {
   /// \param node_id The specified node id.
   void OnNodeDead(const NodeID &node_id);
 
+  /// Mark the manager that there's no more placement group scheduling going on.
+  void MarkSchedulingDone() { scheduling_in_progress_id_ = PlacementGroupID::Nil(); }
```

Comment:
I don't think we should make this method just to add a test scenario. This breaks the abstraction barrier (and shouldn't be a public method).

Reply:
It's not a good enough way, but I have no better solution for the time being, do you have any better suggestion? or, we can add some comment temporarily.